tests/Exception/UnsupportedFilterableExceptionTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Exception;
+
+use Lmc\ApiFilter\AbstractTestCase;
+
+class UnsupportedFilterableExceptionTest extends AbstractTestCase
+{
+    /**
+     * @test
+     */
+    public function shouldBeCatchableByInstances(): void
+    {
+        $expectedExceptions = [
+            \Throwable::class,
+            \Throwable::class,
```

Comment:
One `\Throwable::class` should be enough. Same in other test cases.

Reply:
ups my bad.. I had missed a `skip` keyword in coding style configuration, so the fixer fixed `\Exception` to `\Throwable` :+1: